rulesets/guardian-securedrop-ruleset.xml

```diff
@@ -1,5 +1,5 @@
 <ruleset name="The Guardian">
 	<target host="theguardian.securedrop.tor.onion" />
 	<rule from="^http[s]?://theguardian.securedrop.tor.onion"
-    to="http://33y6fjyhs3phzfjj.onion" />
+    to="http://xp44cagis447k3lpb4wwhcqukix6cgqokbuys24vmxmbzmaq2gjvc2yd.onion/" />
```

Comment:
@emkll mentioned out of band that the trailing slash here is incorrect, we'll need to regenerate (and re-sign).